src/templates/components/CapiHostedCard.svelte

```diff
@@ -0,0 +1,131 @@
+<script context="module" lang="ts">
+	import { clickMacro } from '$lib/gam';
+</script>
+
+<script lang="ts">
+	import AudioIcon from './icons/AudioIcon.svelte';
+	import CameraIcon from './icons/CameraIcon.svelte';
+	import VideoIcon from './icons/VideoIcon.svelte';
+
+	export let card;
+
+	const { headline, image, url, audioTag, galleryTag, videoTag } = card;
+
+	const pictureSupported =
+		image?.sources?.length > 0 && 'srcset' in new Image();
+</script>
+
+<a href={clickMacro(url)}>
```

Comment:
We also need to have `target="_top"` here (as I've learnt the hard way!)

Reply:
@emma-imber I wonder if all templates should have `target="_top"` or we should follow how it works in legacy? I am asking because in Fabric web in legacy it opens in a new tab so I added `target="_blank"`

Reply:
I've personally followed the legacy behaviour - legacy CAPI templates open links in the same window so `target="_top"` is the appropriate choice. If `target="_blank"` is the behaviour of the legacy fabric template then I would stick with that!